samples/OpenStore365.Samples.Serverless.Auth/Functions/GetUserClaimHttpHeaders.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using OpenStore365.Serverless;
+using System.Linq;
+
+namespace OpenStore365.Samples.Serverless.Auth.Functions
+{
+    public static class GetUserClaimHttpHeaders
+    {
+        [FunctionName(nameof(GetUserClaimHttpHeaders))]
+        public static IActionResult Execute(
+            [HttpTrigger(AuthorizationLevel.Anonymous, HTTP.GET, Route = "Http/Headers/UserClaims")] HttpRequest request) =>
+                new OkObjectResult(request.Headers
+                    .Where(x => x.Key.StartsWith("X-MS-CLIENT"))
+                    .ToList());
```

Comment:
Wyjaśnij co to robi pls

Reply:
When we use EasyAuth in order to authenticate incoming HTTP request prior to the request hitting the Azure Function code, EasyAuth authenticates the OAuth2.0 token attached to the request with AAD B2C instance and if it's valid, it extracts all the user claims and appends them to the original HTTP request headers with `X-MS-CLIENT` prefix. This way you can read user claims inside Azure Function code by reading the HTTP headers of the request.

Reply:
I'll explain it with details during next meeting. It's easier to understand with Postman example.